stages/org.osbuild.ostree.remotes

```diff
@@ -113,6 +141,14 @@ def main(tree, options):
                    repo=target,
                    _input=key)
 
+        for cookie in cookies:
+            domain = cookie["domain"]
+            path = cookie["path"]
+            cookie_name = cookie["cookie_name"]
+            cookie_value = cookie["value"]
```

Comment:
Should these be marked as required in the schema for the item?

Reply:
Yes I think so, they're all required.